[2022-04-13 22:35:09,055] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-03-02T06:00:32+00:00 [queued]>
[2022-04-13 22:35:09,086] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-03-02T06:00:32+00:00 [queued]>
[2022-04-13 22:35:09,087] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:35:09,087] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-13 22:35:09,088] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:35:09,126] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-03-02 06:00:32+00:00
[2022-04-13 22:35:09,132] {standard_task_runner.py:52} INFO - Started process 10822 to run task
[2022-04-13 22:35:09,139] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-03-02T06:00:32+00:00', '--job-id', '1841', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpcctk6y2d', '--error-file', '/tmp/tmpy8xgbhr0']
[2022-04-13 22:35:09,144] {standard_task_runner.py:77} INFO - Job 1841: Subtask ingest
[2022-04-13 22:35:09,271] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-03-02T06:00:32+00:00 [running]> on host 13db72185839
[2022-04-13 22:35:09,381] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-03-02T06:00:32+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-02T06:00:32+00:00
[2022-04-13 22:35:09,383] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:35:09,385] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:35:09,395] {subprocess.py:85} INFO - Output:
[2022-04-13 22:35:09,402] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:35:09,403] {subprocess.py:89} INFO - dags
[2022-04-13 22:35:09,403] {subprocess.py:89} INFO - logs
[2022-04-13 22:35:09,405] {subprocess.py:89} INFO - output.csv
[2022-04-13 22:35:09,406] {subprocess.py:89} INFO - plugins
[2022-04-13 22:35:09,406] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:35:09,407] {subprocess.py:89} INFO - scripts
[2022-04-13 22:35:09,407] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:35:09,408] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:35:09,453] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220302T060032, start_date=20220413T223509, end_date=20220413T223509
[2022-04-13 22:35:09,513] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:35:09,567] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
